info/description.md

```diff
@@ -68,7 +67,7 @@ from battle import ROLE
 
 - `unit` - Mobile fighting items. Appears from crafts. `ROLE.UNIT`
 - `tower` - Stationary fighting items. `ROLE.TOWER`
-- `center` - Command Centers. The main building. If it's destroyed, then a battle is over. `ROLE.CENTER`
+- `center` - Command Centers. The ain building. If it's destroyed, then the battle is over. `ROLE.CENTER`

```

Comment:
It's on line 108. Sorry if that's not what you are looking at. I don't really understand how this comment system works.